scripts/uploader.rb

```diff
@@ -8,9 +8,14 @@ class Uploader
     attr_reader :container, :directory, :type
 
     NIGHTLY_BUILD_RETENTION_TIME = 8.weeks
+    CONFIG_FILE = Pathname.new(__dir__).join("../config/upload.yml")
 
     def self.upload(directory, type)
-      new(directory, type).run
+      if File.exist?(CONFIG_FILE)
```

Comment:
FWIW, once you have a Pathname you can call most File.xxx methods on it; this could be `CONFIG_FILE.exists?`

Reply:
oh yeah, true... oops...